client/post-editor/editor-ground-control/index.jsx

```diff
@@ -134,6 +136,24 @@ export default React.createClass( {
 		} );
 	},
 
+	showDateWarning( date ) {
+		const { savedPost, warnPublishDateChange } = this.props;
+
+		if ( ! savedPost ) {
+			return;
+		}
+
+		const currentDate = this.moment( date );
+		const ModifiedDate = this.moment( savedPost.date );
+		const diff = !! currentDate.diff( ModifiedDate );
+
+		if ( savedPost.type !== 'page' && postUtils.isPublished( savedPost ) && diff ) {
```

Comment:
Oh it looks like posts are the only ones that have date based links? I tried a few cpts and they didn't appear to have this issue.